src/ContentValidationListenerFactory.php

```diff
@@ -10,12 +10,14 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ContentValidationListenerFactory
+class ContentValidationListenerFactory implements FactoryInterface
```

Comment:
:+1:  The Factory interface is needed otherwise zend-servicemanager 2.x doesn't deal with the factory correctly. SM 3.x is completely fine with this class currently, and will also be fine with the interface thanks to the migration support in SM 3.x.